### PR TITLE
Monkeypatch suds for json output

### PR DESCRIPTION
**Pull request by fwiesel**
Suds prints its objects over multiple lines which
results in a logging message being broken over lines
whereever such an object is passed as value.

By printing it as a json we can represent the value
in a single line

### CODE REVIEW

**Review by joker-at-work — COMMENTED** [review]
Could you fix the pep8 checks, please?

**Comment by fwiesel** [comment]
I do not see any failures on the files I touched with the version specified in rocky (2.5.5 (pep8: 1.7.1, mccabe: 0.4.0, pyflakes: 1.0.0) CPython 2.7.18 on Linux)

**Comment by joker-at-work** [comment]
```
./oslo_vmware/_monkeypatch.py:1:1: H102  Apache 2.0 license header not found
import suds.sudsobject
^
./oslo_vmware/_monkeypatch.py:2:1: H306  imports not in alphabetical order (suds.sudsobject, json)
import json
^
./oslo_vmware/_monkeypatch.py:3:1: H306  imports not in alphabetical order (json, ..vim_util)
from . import vim_util
^
./oslo_vmware/_monkeypatch.py:3:6: H304  No relative imports. 'from . import vim_util' is a relative import
from . import vim_util
     ^
./oslo_vmware/_monkeypatch.py:6:1: H238  old style class declaration, use new style (inherit from `object`)
class _JsonPrinter:
^
```

File: oslo_vmware/_monkeypatch.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+import suds.sudsobject
+import vim_util
```
Comment (joker-at-work) [inline oslo_vmware/_monkeypatch.py:18]: Shouldn't this be `from oslo_vmware import vim_util`?

Comment (fwiesel) [reply oslo_vmware/_monkeypatch.py]: True, I've changed it. `import vim_util` works too for some reason.

**Review by joker-at-work — APPROVED** [review]

**Pull request merged**